napi_from_js.md

```diff
@@ -365,3 +257,77 @@ status = napi_new_instance(env, constructor, argc, argv, &value);
 
 #### Return value
 - `napi_ok` if the API succeeded.
+
+### *napi_make_callback*
+
+#### Signature
+```C
+napi_status napi_make_callback(napi_env env,
+                               napi_value recv,
+                               napi_value func,
+                               int argc,
+                               const napi_value* argv,
+                               napi_value* result)
+```
+
+#### Parameters
+- `[in]  env `: The environment that the API is invoked under
+- `[in]  recv`: The `this` object passed to the called function
+- `[in]  func`: `napi_value` representing the JavaScript function
+to be invoked
+- `[in]  argc`: The count of elements in the `argv` array
+- `[in]  argv`: Array of JavaScript values as `napi_value` 
+representing the arguments to the function. If there are more arguments
+than the provided argument count, then only the requested number of 
+arguments are copied. If there are fewer arguments provided than 
+claimed, the rest of `argv` is filled with `napi_value` values that 
+represent `undefined`
```

Comment:
It looks like this description of `argv` was copy-passed from `napi_get_cb_info()`. It doesn't apply here.